src/components/filters/statusFilter.tsx

```diff
@@ -0,0 +1,56 @@
+import arrayOfTasks from 'arrayOfTasks'
+import React, { useState } from 'react'
+//utils
```

Comment:
I liked this component, but should place it in the ```domain/tasks``` folder. . ```components``` folder is meant to have only components which can be used in any or several parts of the app. The status filters is only ment to be used in the ```tasks``` screen